backend/yarn.lock

```diff
@@ -0,0 +1,655 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This is a pretty big dependency closure to bring in just to run `npm run lint-staged` instead of `gradlew spotlessApply`